ManatiCR/vitaminau
# VIT-46: Update "Trabajo en curso" form

# Task [VIT-46](https://manati.atlassian.net/browse/VIT-46?atlOrigin=eyJpIjoiZGY0NGIwNjMwODg3NDVmMzgwMGUyZjc2ZTljMjViMTgiLCJwIjoiaiJ9)
Update of the "Trabajo en Curso" form.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following url: 
  - http://vitaminau.docker/form/work-in-progress
- Once on this page, go to the "Actividad" section.
- Try pressing the "+" button to add a new row of items.
- Try pressing the "-" button to delete a row of items.
- Verify that the changed field is the same as the Jira ticket.
  - [VIT-46](https://manati.atlassian.net/browse/VIT-46?atlOrigin=eyJpIjoiZGY0NGIwNjMwODg3NDVmMzgwMGUyZjc2ZTljMjViMTgiLCJwIjoiaiJ9)
- Fill all the fields of the form and press the save buttom
- Once you press it your information should be saved and the form should redirect you to a thank you page

# Acceptance criteria
- I can add multiple data sets.
- I can add it and I have no errors.
- The help text of the "Beneficiarios" fieldset must be "Indique los beneficiarios de todas las actividades del trabajo de curso."
- The help text of the "Número de beneficiarios esperados" field must be "Indique la cantidad de beneficiarios de esta actividad."
